### PR TITLE
Category title should be non-clickable on the homepage 

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by smarzban** [comment]
@miltontony Small review please :)

File: unicorecmsconnectsmart/templates/home.pt [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
         <div class="articles category-${cat.slug}" tal:repeat="cat view.get_categories()">
             <div class="h1">
-                <a href="/content/list/${cat.uuid}/">${cat.title}</a>
+                <a>${cat.title}</a>
```
Comment (miltontony) [inline unicorecmsconnectsmart/templates/home.pt]: Just remove the `<a>` completely leaving `${cat.title}`

**Comment by miltontony** [comment]
otherwise :+1:

**Comment by smarzban** [comment]
@miltontony tiny review please :)

**Comment by miltontony** [comment]
:+1:

**Pull request merged**